### ISSUE TITLE
"Degree"-values (Input forms) are transparent

### ISSUE THREAD

**Issue by el-Presi** [HerrRiebmann/Caravan_Leveler#3]
Hi!

Thanks first of all for this cool sketch here!
It works so far on my side, but I got one issue. The Value boxes or input forms are quiet transparent here, so its hard to read the value (when the sun is shining onto the smartphone).
I'm not an expert in CSS and stuff and I do not get it managed to have this solid filled with red to green (depending on value).
Here you can see my version - I use it on iOS.
Compared to your screenshot in the readme, it is very hard to read my values here.
![IMG_7415](https://github.com/HerrRiebmann/Caravan_Leveler/assets/47759077/3f5c8b9b-7961-4be1-908f-1250a0c8a9bf)

Hope someone has an idea! :) Thanks!!

**Comment by HerrRiebmann** [comment]
Hi, very nice to see another image in the background an some changed colors :)
Honestly I have not done very much with this project in the past.
Working on different things leaded to some improvements which I´d merge back into this one.
Maybe it makes sense to make the font much bigger and make it easier to adjust the colors for each element.

Then I think it also would be a bit more secure to add a WiFi password and/or only allow changes on the settings with a PIN or something.
I´ll try to throw an eye on this during the week.
Thanks for the feedback and for putting back this project into my focus!

**Comment by HerrRiebmann** [comment]
Hi @el-Presi 
the easiest workaround is to enlarge the values by adding this to the CSS file:
```
input[type="text"]
{
    font-size:20px;
}
```
And/Or setting the text to bold. 

From my code I cannot see why the values (which were input boxes) have an opacity. This might be an iPhone thing?
The Color is calculated by hsl. Using the first value from 0-120 is red to green. Second parameter is saturation (100) and last brightness (50). Which also does not make anything transparent.
You can test it here, if you´d like: [CSS HSL Colors](https://www.w3schools.com/css/css_colors_hsl.asp)

On my computer it´s not transparent as well. I need to grab another ESP to test it on my phone.

**Comment by el-Presi** [comment]
Yes, it really seems to be an iPhone issue, since on the PC/Mac just by viewing the website, all is fine..

**Comment by HerrRiebmann** [comment]
You might overwrite that by setting the input background to white?
```
input[type="text"]
{
    font-size:20px;
    background-color: white; 
}
```

Found my old iPhone and try to make it work again...
